crates/bevy_audio/src/lib.rs

```diff
@@ -19,6 +19,108 @@
 //!     });
 //! }
 //! ```
+//!
+//! # Contributing
```

Comment:
This is user-facing documentation. This is likely more suited for a crate level CONTRIBUTING.md, or as a non-doc comment. The overall design philosophy is probably worth mentioning in passing or tucking away in a public but lower level module, but much of this are implementation details the average user shouldn't need to think about when using the engine.

Reply:
Just saw your comment after pushing some changes. I do agree with this, putting it in the crate-level doc comments was something @alice-i-cecile suggested, but any place for it is fine by me. `CONTRIBUTING.md` is a good alternative place I think.